.github/workflows/nodejs.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         node-version: ${{ matrix.node-version }}
     - name: npm install, build, and test
       run: |
-        npm install -g @angular/cli@1.4.10
+        npm install -g @angular/cli@1.7.4
```

Comment:
Why is this changed and what is the impact of this for user?

Reply:
I have reverted this change.  This change was made as there was an open Mr by GitHub bot to upgrade the dependency `@angular/cli@1.4.10`. On testing with upgraded ci SDK, it worked fine, but this change could have restricted users with `@angular/cli@1.7.4`.